openapi/openapi.json

```
@@ -32663,6 +32702,26 @@
         }
       }
     },
+    "com.github.openshift.api.operator.v1.Theme": {
+      "description": "Theme defines a theme type for the console UI.",
+      "type": "object",
+      "required": [
+        "type",
+        "file"
+      ],
+      "properties": {
+        "file": {
+          "description": "file is a required field that references a specific file within a ConfigMap in the openshift-config namespace. This field allows the console to locate and use the specified file containing a custom logo. You can create it with a command like: - 'oc create configmap custom-logos-config --namespace=openshift-config --from-file=/path/to/file The ConfigMap key should include the file extension for the dark or light or default theme so that the console serves the file with the correct MIME type. Recommended Masthead and About Modal logo specifications: Image size must be less than 1 MB due to constraints on the ConfigMap size Dimensions: Max height of 60px and max width of 200px SVG format preferred Recommended Favicon specifications: Dimensions: Max height of 16px and max width of 16px PNG format preferred",
```

```suggestion
          "description": "file is a required field that references a specific file within a ConfigMap in the openshift-config namespace. This field allows the console to locate and use the specified file containing a custom logo. You can create it with a command like:  'oc create configmap custom-logos-config --namespace=openshift-config --from-file=/path/to/file'. The ConfigMap key must include the file extension for the dark or light or default theme so the console serves the file with the correct MIME type. Recommended Masthead and About Modal logo specifications: Image size must be less than 1 MB due to constraints on the ConfigMap size Dimensions: Max height of 60px and max width of 200px SVG format preferred. Recommended Favicon specifications: Dimensions: Max height of 16px and max width of 16px PNG format preferred",
```

should that should be a must?

Also, I may be wrong about the command syntax but should there be another ' at the end?

---

I think that since the key is required for the logo file, the 'must' is appropriate. cc: @jhadvig